### PR TITLE
feat: add new protocols to defillama asset factory

### PR DESCRIPTION
**Pull request by evanameyer1**

### CODE REVIEW

**Review by ryscheng — COMMENTED** [review]
Did you use isort somewhere?

**Comment by evanameyer1** [comment]
> Did you use isort somewhere?

All I did was add the new s6 projects into the list of defillama protocols. When I was committing the code I ran into an issue by not having isort downloaded I believe, so I pip installed it.

**Comment by ryscheng** [comment]
Ah sounds good! Let's merge. Do you know how to run it?

**Comment by evanameyer1** [comment]
> Ah sounds good! Let's merge. Do you know how to run it?

I think so, Javi said I go to http://admin-dagster.opensource.observer, search for the asset and click “materialize”, correct?

File: warehouse/oso_dagster/assets/defillama.py [hunk 1]
```diff
@@ -7,7 +7,34 @@
...
     "aave-v3",
     "optimism-bridge",
-    # ...
+    # Old protocols...
+    "ether.fi",
+    "compound-finance",
+    "dforce",
+    "alchemix",
+    "toros",
+    "lisk",
```
Comment (Jabolol) [inline warehouse/oso_dagster/assets/defillama.py:16]: This protocol `lisk` does not exist on DefiLLama

File: warehouse/oso_dagster/assets/defillama.py [hunk 2]
```diff
@@ -7,7 +7,34 @@
...
+    "derive",
+    "contango",
+    "clusters",
+    "kelp-dao",
+    "lets-get-hai",
+    "beefy",
+    "ionic-protocol",
+    "resolv",
+    "pyth-network",
+    "kroma",
```
Comment (Jabolol) [inline warehouse/oso_dagster/assets/defillama.py:27]: `kroma` is not a valid slug either

File: warehouse/oso_dagster/assets/defillama.py [hunk 3]
```diff
@@ -7,7 +7,34 @@
...
+    "tlx-finance",
+    "uniswap",
+    "velodrome",
+    "origin",
```
Comment (Jabolol) [inline warehouse/oso_dagster/assets/defillama.py:31]: `origin` is not a valid slug

File: warehouse/oso_dagster/assets/defillama.py [hunk 4]
```diff
@@ -7,7 +7,34 @@
...
+    "origin-protocol",
+    "polynomial-protocol",
+    "extra-finance",
+    "frax-finance",
+    "mode",
```
Comment (Jabolol) [inline warehouse/oso_dagster/assets/defillama.py:36]: `mode` is not a valid slug

**Review by Jabolol — COMMENTED** [review]
Looks good to me, but please check that the protocols are valid on `DefiLLama`. API docs [`here`](https://defillama.com/docs/api). I marked the invalid ones on the review comments 🫶🏻

**Pull request merged**